content/en/docs/instrumentation/net/shim.md

```diff
@@ -312,3 +312,12 @@ using var span = tracer.StartActiveSpan("another-span", links: links);
 
 // do some work
 ```
+
+## Next steps
+
+If you're not utilizing [instrumentation libraries]({{< relref "automatic" >}}), it's highly recommended that you do so.
+Instrumentation libraries will automatically instrument relevant libraries you're using and generate
+data for things like inbound and outbound HTTP requests and more.
+
+You'll also want to configure an appropriate exporter to [export your telemetry data]({{< relref "exporters" >}})
+to one or more telemetry backends.
```

Comment:
As a general "markdown style" comment that applies to all new content that you're adding: please line-wrap paragraphs to 80 columns (if you're using VS Code, there's a plugin that makes it easy to re-wrap paragraphs). Thanks!

Reply:
Will be addressed through followup PRs like #1072.